frontend/src/lib/utils.tsx

```diff
@@ -2157,10 +2157,3 @@ export function getRelativeNextPath(nextPath: string | null | undefined, locatio
         return null
     }
 }
-
-export function escapeHtml(raw: string): string {
```

Comment:
Not sure if you intended to use this somewhere else @Piccirello. I removed it because there's no usage right now, feel free to add it back if you ever need it. It's unusual to need something like this, though, since React can handle this stuff just fine (like you mentioned in your previous PR)

Reply:
With `dangerouslySetInnerHTML` removed, we don't really have much use for `escapeHTML` any more. This lgtm 👍